### PR TITLE
Remove author and description from pixi.toml

### PR DESCRIPTION
**Pull request by Hofer-Julian**
Probably best to leave them out, instead of having wrong ones :)

### CODE REVIEW

File: pixi.toml [hunk 1]
```diff
@@ -1,8 +1,6 @@
 [project]
 name = "lfortran"
 version = "0.1.0"
-description = "Add a short description here"
-authors = ["Wolf Vollprecht <w.vollprecht@gmail.com>"]
```
Comment (certik) [inline pixi.toml:5]: Let's use "LFortran Maintainers <lfortran@groups.io>" for authors, and "Modern interactive LLVM-based Fortran compiler" for description.

**Comment by certik** [comment]
@Hofer-Julian thanks! I left a comment how to fix this.

File: pixi.toml (hunk 1 above)
Comment (Hofer-Julian) [reply pixi.toml:5]: Done

**Review by certik — APPROVED** [review]
Thanks!

**Pull request merged**